docs/development/release.md

```diff
@@ -82,19 +92,25 @@ If you created a new release plan in step one, you will need to create a new bam
 
 #### Creating a Bamboo Deployment plan
 
-* In the Cumulus Core project (<https://ci.earthdata.nasa.gov/browse/CUM-CBA>), click Actions -> Configure Plan
+* In the Cumulus Core project (<https://ci.earthdata.nasa.gov/browse/CUM-CBA>), click `Actions -> Configure Plan` in the top right.
+
+* Scroll to the bottom of the branch list in the bottom left and click `Create Plan Branch`.
+
+* Click `Create plan branch manually`.
 
-* Scroll to the bottom of the branch list and click `Create Plan Branch`
+* Add the values in that list. Choose a display name that makes it *very* clear this is a deployment branch plan. `Release (branch name)` seems to work well. *Make sure* you enter the correct branch name.
 
-* Click `Create plan branch manually`
+* **Important** Deselect Enable Branch - if you do not do this, it will immediately fire off a build.
 
-* Add the values in that list.   Choose a display name that makes it *very* clear this is a deployment branch plan.    `Release (branch name)` seems to work well.    *Make sure* you select the correct branch
+* **Do Immediately** On the `Branch Details` page, enable `Change trigger`.  Set the `Trigger type` to manual, this will prevent commits to the branch from triggering the build plan.
+You should have been redirected to the `Branch Details` tab after creating the plan. If not, navigate to the branch from the list where you clicked `Create Plan Branch` in the previous step.
 
-* **Important** Deselect Enable Branch - if you do not do this, it will immediately fire off a build
+* Go to the `Variables` tab and set GIT_PR, USE_NPM_PACKAGES, SKIP_AUDIT and PUBLISH_FLAG to true. Ensure that you are on your branch plan and not the `master` plan: You should not see a large list of configured variables, but instead a dropdown allowing you to select variables to override, and the tab title will be `Branch Variables`.
```

Comment:
This should be USE_TERRAFORM_ZIPS.  If this wasn't set, you didn't run integration tests against the deployed package :S

Reply:
There are 4 flags listed here. Which one is incorrect?

Reply:
I've found `USE_NPM_PACKAGES` is no longer evaluated.